packages/docs/src/examples/combobox/advanced.vue

```diff
@@ -25,8 +25,7 @@
     </template>
     <template
       v-if="item === Object(item)"
```

Comment:
this is throwing errors 
![image](https://user-images.githubusercontent.com/9064066/52865911-3d065980-3103-11e9-8783-e5b3de4446c3.png)